src/adminDashboard/adminDashboard.ts

```diff
@@ -1,19 +1,26 @@
-import { EthereumService } from "services/EthereumService";
-import { Seed } from "entities/Seed";
+import { TransactionReceipt } from 'services/TransactionsService';
+import { Seed } from './../entities/Seed';
+import { Address, EthereumService } from "services/EthereumService";
 import { autoinject, computedFrom } from "aurelia-framework";
 import { SeedService } from "services/SeedService";
 import "./adminDashboard.scss";
 import { EventAggregator } from "aurelia-event-aggregator";
 import { DisposableCollection } from "services/DisposableCollection";
 import { EventConfigException } from "services/GeneralEvents";
-import { Utils } from "services/utils";
+import { WhiteListService } from 'services/WhiteListService';
+import axios from 'axios';
+import TransactionsService from 'services/TransactionsService';
 
 @autoinject
 export class AdminDashboard {
 
   seeds: Array<Seed> = [];
+  selectedSeed: Seed;
+  addressToRemove: String = '';
+  addressToAdd: String = '';
+  receiverAddress: String = '';
   subscriptions: DisposableCollection = new DisposableCollection();
-  loading = true;
+  loading: boolean = true;
```

Comment:
just FYI, `loading = true` is fine because typescript infers the datatype from the assignment.  but definitely doesn't hurt to be explicit.

Reply:
ohk. I haven't changed it, should I remove the type `boolean`